paddle/fluid/framework/details/inplace_op_pass.cc

```diff
@@ -342,6 +380,97 @@ void InplacePass::TryInplaceOpInputOutput(ir::Node* op,
   }
 }
 
+void GraphView::TopoSort(ir::Graph* graph) {
```

Comment:
Need performance comparison between previous `SortLike` and current `TopoSort`.

Reply:
I can provide a report later